src/style.css

```diff
@@ -0,0 +1,53 @@
+html,
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  height: 14px;
+  width: 16px;
+  margin-top: 2px;
+  margin-left: 2px;
+  margin-right: 2px;
```

Comment:
It's gonna be `margin-right: 4px` for all elements besides last one (use `:not(:last-child)` )